src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx

```diff
@@ -352,6 +353,13 @@ export function InteractiveTutorialRoot(): React.ReactElement {
             continuously hack the n00dles server.
             <br />
             <br />
+            To access{" "}
+            <Typography component="span" color={Settings.theme.warning}>
```

Comment:
I think either this shouldn't be colored differently, or you should go all the way and make this a `<Link>` too.